assets/blocks/button/button.editor.scss

```diff
@@ -1,4 +1,4 @@
-.wp-block .wp-block-sensei-button .wp-block-button__link:disabled {
+.block-editor-block-styles .wp-block .wp-block-sensei-button .wp-block-button__link:disabled {
```

Comment:
It's not needed for this PR, but it just makes it more specific for the style's thumbnails, so we can use disabled buttons in the editor if we need.